docs-src/cli/task-queue/get-build-id-reachability.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Add the new `temporal task-queue` subcommands (`get-build-id-reachability`, `get-build-ids`, and `update-build-ids`) to the `task-queue.json` config file.